Docs/source/running_cpp/parameters.rst

```diff
@@ -1023,6 +1023,11 @@ Numerics and algorithms
     Note that the implementation in WarpX is more efficient when these 3 numbers are equal,
     and when they are between 1 and 3.
 
+* ``interpolation.match_shape_factors`` (`0` or `1` ; default: `0`)
+    On a staggered grid, default shape factors are different for different field components.
```

Comment:
```suggestion
    On a staggered grid, default particle shape factors are different for different field components.
```

Reply:
Admittedly, I do not understand what this sentence tries to say.
Yes, if `nox` et al. are individually set, then the shape factor is different per direction... do you mean something solver specific?

Reply:
What I mean is that when we interpolate the fields to the particles, we sometimes use different interpolation for e.g. E_x and E_z. This PR doesn't force the shape factors to be the same along all directions, but it does force the interpolation to be the same for all field components.